api_tests/src/test_environment.ts

```diff
@@ -259,7 +259,12 @@ export default class TestEnvironment extends NodeEnvironment {
     private async setupLightningChannels() {
         const { alice, bob } = this.global.lndWallets;
 
-        await alice.connectPeer(bob);
+        const alicePeers = await alice.listPeers();
+        const bobPubkey = await bob.inner.getPubkey();
+
+        if (!alicePeers.find((peer) => peer.pubKey === bobPubkey)) {
```

Comment:
Could have done this check inside the fn to make it idempotent :)

Anyway, thanks for the fix! <3